docs/index.md

```diff
@@ -85,5 +83,7 @@ api_reference/user_quotas
 api_reference/troubleshooting
 api_reference/faq
 api_reference/glossary
+api_reference/best-practices
```

Comment:
This is a nice idea and we can keep the file placeholder, but let's omit it from the index to not be rendered since we may deploy soon the current development branch.

Reply:
Sounds good! I'll remove them